rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/server/RMServiceImpl.java

```diff
@@ -474,6 +492,27 @@ private String executeFunctionReturnStreamAsString(Function<RestClient, InputStr
         }
     }
 
+    private String executeFunctionReturnStreamAsString(Function<RestClient, InputStream> function)
+            throws ServiceException, RestServerException {
+        RestClient restClientProxy = getRestClientProxy();
+
+        InputStream inputStream = null;
+
+        try {
+            inputStream = function.apply(restClientProxy);
+
+            try {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Extract this nested try block into a separate method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1141)